glusterd2/commands/volumes/common.go

```diff
@@ -380,3 +419,25 @@ func txnGenerateBrickVolfiles(c transaction.TxnCtx) error {
 	}
 	return nil
 }
+
+func containsReservedGroupProfile(opts interface{}) bool {
+	pfx := "profile.default."
+	switch value := opts.(type) {
+	case map[string]string:
+		for k := range value {
+			if strings.HasPrefix(k, pfx) {
+				return true
+			}
+		}
+	case []string:
+		for _, v := range value {
+			if strings.HasPrefix(v, pfx) {
+				return true
+			}
+		}
+	default:
```

Comment:
can we discard  default, as we are returning false at the end

Reply:
ok

Reply:
done